src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -1,103 +1,122 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-	<!-- 
-		Microsoft ResX Schema
+  <!-- 
+    Microsoft ResX Schema 
```

Comment:
Not sure why this changed.  All i did was use the normal VS designer to edit these files.